contract/models/contract.py

```diff
@@ -177,6 +177,18 @@ def _compute_access_url(self):
         for record in self:
             record.access_url = "/my/contracts/{}".format(record.id)
 
+    def _get_share_url(
+        self, redirect=False, signup_partner=False, pid=None, share_token=True
+    ):
+        if redirect:
```

Comment:
I don't think both arguments are related, so this seems more a trick than the real solution.

Reply:
I have removed `_get_share_url` function and i have used other trick to solve PR description.